src/organisms/Filter/Filter.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  ChangeEvent,
+  FC,
+  KeyboardEvent,
+  ReactNode,
+  useRef,
+  useState,
+} from 'react';
+
+import { Button, Icon } from '@equinor/eds-core-react';
+import {
+  arrow_drop_down,
+  arrow_drop_up,
+  clear,
+  filter_list,
+  IconData,
+} from '@equinor/eds-icons';
+
+import { Container, Content, StyledChip, Wrapper } from './Filter.styles';
+import { colors } from 'src/atoms/style/colors';
+
+import { AnimatePresence } from 'framer-motion';
+
+export interface FilterProps {
+  values: { value: string; label: string; icon?: IconData }[];
+  onClearFilter: (value: string) => void;
+  onClearAllFilters: () => void;
+  onSearch: (value: string) => void;
+  children: ReactNode | ReactNode[];
+  initialOpen?: boolean;
+  placeholder?: string;
+  showClearFiltersButton?: boolean;
+}
+
+export const Filter: FC<FilterProps> = ({
+  values,
+  onSearch,
+  onClearFilter,
+  onClearAllFilters,
+  children,
+  initialOpen = false,
+  placeholder = 'Search...',
+  showClearFiltersButton = true,
```

Comment:
We should have JSDoc descriptions for the props i think

Reply:
Oh shiz, I thought I had that already